index.js

```diff
@@ -4,22 +4,33 @@ const timeout = require('timeout-refresh')
 const inspect = require('inspect-custom-symbol')
 const Nanoguard = require('nanoguard')
 const pretty = require('pretty-hash')
+const { Message } = require('message-pair')
 const { Duplex } = require('streamx')
 
+class StreamExtension extends Message {
+  send (message) {
+    const stream = this.pair.handlers
+    if (stream._changes !== this.pair.changes) {
```

Comment:
Can you explain the `_changes` property a bit?

Reply:
the changes is a simple counter that increments when the extensions manager has been updated